net/ietf/message.d

```diff
@@ -37,7 +37,6 @@ import ae.utils.time;
 
 import ae.net.ietf.wrap;
 
-alias ae.utils.text.ascii.ascii ascii; // https://d.puremagic.com/issues/show_bug.cgi?id=12156
```

Comment:
This line should be an error but I didn't fix it because the alias seems no longer needed, `ascii` is already available given the `import ae.utils.text;` a little above.
I'm not sure if that is going to break building with a very old compiler.